### ISSUE TITLE
Option to translate @room to @here

### ISSUE THREAD

**Issue by auscompgeek** [matrix-org/matrix-appservice-slack#327]
The Discord bridge translates `@room` to `@here`. It'd be nice to have that option with the Slack bridge too.

**Comment by Half-Shot** [comment]
You want this to be configurable ? https://github.com/matrix-org/matrix-appservice-slack/blob/54505157d20137e7a0b0443c5c9698f269a30f0f/src/substitutions.ts#L161

**Comment by auscompgeek** [comment]
Yes. Preferably per-room as well (so it'd be usable on the matrix.org hosted bridge).
